docs/fundamentals/monorepo.md

```diff
@@ -0,0 +1,296 @@
+# Building like Google: Monorepos and Automation
+
+In this guide you will look at one of the most interesting parts of Nx. The part that can make so many things so much easier, that it can have a transformative effect on a team and even on an organization.
```

Comment:
This sounds suspenseful to me.

I think it should be more explicit that a Monorepo has these effects.